transform/snowflake-dbt/models/blapi/customers_with_freemium_subs_without_filter.sql

```diff
@@ -0,0 +1,159 @@
+{{config({
```

Comment:
add a comment on how this file differs from `customers_with_freemium_subs.sql` ?

Reply:
I have removed this model entirely. Added comment for the new one.